openmls/src/group/errors.rs

```diff
@@ -107,15 +107,9 @@ pub enum ExternalCommitError<StorageError> {
     /// We don't support the ciphersuite of the group we are trying to join.
     #[error("We don't support the ciphersuite of the group we are trying to join.")]
     UnsupportedCiphersuite,
-    /// Sender not found in tree.
```

Comment:
Enum changes are breaking changes. So these need to go into the changelog.

Reply:
Good point. I reverted all changes to error enums.